examples/backports.pp

```diff
@@ -1 +1,32 @@
-# TODO
+$location = $::apt::distcodename ? {
+  'squeeze' => 'http://backports.debian.org/debian-backports',
+  'wheezy'  => 'http://ftp.debian.org/debian/',
+  default   => 'http://us.archive.ubuntu.com/ubuntu',
```

Comment:
Use `archive.ubuntu.com`, without the `us.` prefix. That should give people a geographically more desirable mirror.